tests/integration/tab/test_tabs_with_graphs.py

```diff
@@ -230,6 +230,7 @@ def check_graph_config_shape(dash_dcc):
         'showSources',
         'logging',
         'globalTransforms',
+        'notifyOnLogging',
```

Comment:
@alexcjohnson We already ignore `logging`, seems like we probably want to ignore another `logging` themed prop, but need to be certain.

Reply:
Yes, for now we can ignore this one. In fact `notifyOnLogging` *would* be useful to dash - and alongside it `logging` becomes useful - because this repeats console log messages into the UI as notification boxes. Unfortunately right now this can only be used globally (`Plotly.setPlotConfig`), not per-plot in the `config` prop. In fact the better solution for dash will be when we have logging events https://github.com/plotly/plotly.js/issues/4462, and we can hopefully pass these along to the main dash devtools.